tensorflow/compiler/xla/service/gpu/horizontal_loop_fusion.cc

```diff
@@ -458,22 +564,122 @@ Status HorizontalLoopFusionImpl::CreateFusedComputation(
   return OkStatus();
 }
 
+Status HorizontalLoopFusionImpl::CreateLoopFusedComputation(
```

Comment:
There is a lot of code duplication between "CreateLoopFusedComputation" and "CreateSliceInputFusedComputation", could you reduce that?
(You could either use a method template or even parametrize it).

Reply:
Fixed